docs/source/milestone_managing.md

```diff
@@ -0,0 +1,24 @@
+# Milestone Managing
```

Comment:
I think this (combined with my suggestion on `release.myst`) might fix the docs build issue:
```suggestion
(source/milestone_managing)=
# Milestone Managing
```